### ISSUE TITLE
get_bytes returns scientific notation for very large numbers

### ISSUE THREAD

**Issue by joshku** [wfxr/tmux-net-speed#4]
Hi,

I am on a Raspberry Pi 4B running Ubuntu 21.04 and I found that for very large `rx_bytes` or `tx_bytes` values (gigabyte sizes) the value `cur` in line 30 of `net-speed.sh` gets a value in scientific notation.  https://github.com/wfxr/tmux-net-speed/blob/1fd90d2a436806043ee20fd8403f2a3e21cb5414/scripts/net-speed.sh#L39

I ran `net-speed.sh` in debug mode using bash and you can see the following output

```
++++ echo linux
+++ for file in /sys/class/net/*
+++ [[ /sys/class/net/eth0 =~ .*/(lo|docker.*) ]]
+++ cat /sys/class/net/eth0/statistics/rx_bytes
+++ sum_column
+++ for file in /sys/class/net/*
+++ [[ /sys/class/net/lo =~ .*/(lo|docker.*) ]]
+++ for file in /sys/class/net/*
+++ [[ /sys/class/net/wlan0 =~ .*/(lo|docker.*) ]]
+++ cat /sys/class/net/wlan0/statistics/rx_bytes
++ cur=9.79828e+09
+++ get_tmux_option @netspeed_rx_bytes 9.79828e+09
+++ local option=@netspeed_rx_bytes
+++ local default_value=9.79828e+09
++++ tmux show-option -gqv @netspeed_rx_bytes
+++ local option_value=9798010960
+++ [[ -z 9798010960 ]]
+++ echo 9798010960
++ pre=9798010960
./net-speed.sh: line 41: 9.79828e+09: syntax error: invalid arithmetic operator (error token is ".79828e+09")
```

Looking at the `helpers.sh` code I found that for line 30, if I convert your `print sum;` to `printf "%.0f", sum;` then the output is correct again
https://github.com/wfxr/tmux-net-speed/blob/1fd90d2a436806043ee20fd8403f2a3e21cb5414/scripts/helpers.sh#L30

```
++++ echo linux
+++ for file in /sys/class/net/*
+++ [[ /sys/class/net/eth0 =~ .*/(lo|docker.*) ]]
+++ sum_column
+++ cat /sys/class/net/eth0/statistics/rx_bytes
+++ for file in /sys/class/net/*
+++ [[ /sys/class/net/lo =~ .*/(lo|docker.*) ]]
+++ for file in /sys/class/net/*
+++ [[ /sys/class/net/wlan0 =~ .*/(lo|docker.*) ]]
+++ cat /sys/class/net/wlan0/statistics/rx_bytes
++ cur=9798414757
+++ get_tmux_option @netspeed_rx_bytes 9798414757
+++ local option=@netspeed_rx_bytes
+++ local default_value=9798414757
++++ tmux show-option -gqv @netspeed_rx_bytes
+++ local option_value=9798414757
+++ [[ -z 9798414757 ]]
+++ echo 9798414757
++ pre=9798414757
```

Not sure if this will be the case on other operating systems and devices but I thought I would let you know

**Comment by wfxr** [comment]
I'll look into this when I have some time

**Comment by wfxr** [comment]
@joshku Should have been fixed by #5.

Thanks @joshku @karbobc
